examples/equilibria/eudemo_2017.ex.py

```diff
@@ -300,17 +306,25 @@
 
 sof = deepcopy(reference_eq)
 
+diag_ops = EqDiagnosticOptions(
+    psi_diff=False, split_psi_plots=False, reference_eq=reference_eq
```

Comment:
split_psi_plots is a bool here but this fails when running the notebook on plotting L742